### PR TITLE
Update default reviewers

### PR DESCRIPTION
**Pull request by mairaw**
Part I of our retirement gift to @rpetrusha 🎁

This only sets the default reviewers on the codeowners file.

### CODE REVIEW

**Comment by rpetrusha** [comment]
Thank you, @mairaw!

**Comment by mairaw** [comment]
What are you doing here? 😄 Go relax!

**Review by rpetrusha — APPROVED** [review]
LGTM.

File: .github/CODEOWNERS
Comment (BillWagner) [inline .github/CODEOWNERS]: I've got responsibility for async in C# (along with threading). This is related. Also, happy to work with @gewarren to make sure these areas are seamless.

**Comment by rpetrusha** [comment]
I just happened to be checking email.

**Review by BillWagner — APPROVED** [review]
This LGTM.

I had one question if an area is related.  I'll defer to @gewarren if she wants it changed.

File: .github/CODEOWNERS
Comment (gewarren) [inline .github/CODEOWNERS]:
```suggestion
# Cross platform
```

File: .github/CODEOWNERS
Comment (gewarren) [inline .github/CODEOWNERS]:
```suggestion
/docs/standard/events/** @gewarren
```

File: .github/CODEOWNERS
Comment (gewarren) [inline .github/CODEOWNERS]: design-guidelines folder seems to be missing here.

File: .github/CODEOWNERS
Comment (gewarren) [inline .github/CODEOWNERS]: Add library-guidance folder?

File: .github/CODEOWNERS
Comment (gewarren) [inline .github/CODEOWNERS]: I don't see a linq folder under standard.

File: .github/CODEOWNERS
Comment (gewarren) [inline .github/CODEOWNERS]:
```suggestion
/docs/standard/serialization/** @tdykstra
```

File: .github/CODEOWNERS
Comment (gewarren) [inline .github/CODEOWNERS]: Add native-interop folder?

File: .github/CODEOWNERS
Comment (gewarren) [inline .github/CODEOWNERS]: Put this before WPF to match folder structure.

File: .github/CODEOWNERS
Comment (pkulikov) [inline .github/CODEOWNERS]: This path and the path above are identical

**Review by gewarren — COMMENTED** [review]
There's a handful of folders missing from under docs/framework, but is that just because they'll fall under the umbrella of the parent folder code owner?

File: .github/CODEOWNERS
Comment (mairaw) [reply .github/CODEOWNERS]: It's coming with a vendor task.

File: .github/CODEOWNERS
Comment (mairaw) [reply .github/CODEOWNERS]: I skipped the self-serve folders on this round. Let me add them.

Comment (mairaw) [reply .github/CODEOWNERS]: I put my name for that one. We don't accept PRs for that content (unless you're fixing some kind of formatting or something) since it's copyrighted content.

File: .github/CODEOWNERS
Comment (mairaw) [reply .github/CODEOWNERS]: I didn't add this one since we mark this one for retirement.

File: .github/CODEOWNERS
Comment (mairaw) [reply .github/CODEOWNERS]: I think this makes sense @BillWagner. We should ensure that no one is overwhelmed by the number of PRs.

File: .github/CODEOWNERS
Comment (mairaw) [inline .github/CODEOWNERS]:
```suggestion
/docs/standard/asynchronous-programming-patterns/** @BillWagner
```

**Review by Thraka — APPROVED** [review]

**Pull request merged**
